app/controllers/courses_controller.rb

```diff
@@ -51,6 +51,7 @@ def build_course
       .where(provider_code: @provider_code)
       .find(params[:code])
       .first
+      &.decorate
```

Comment:
I know this is tempting, but this means that when we start to use @course in any of the actions, we won't have a JSON API client resource object, we'll have a decorated object which will behave differently. I think decorated object should only be used in views (or anywhere else explicitly needed), and would rather have an `after_action` hook to decorate, or use `@decorated_course` (which means more changes to views, 🤢 ).

Also: https://github.com/drapergem/draper#when-to-decorate-objects

Reply:
Done